Tools/cases_generator/flags.py

```diff
@@ -176,7 +175,7 @@ def variable_used_unspecialized(node: parsing.Node, name: str) -> bool:
     tokens: list[lx.Token] = []
     skipping = False
     for i, token in enumerate(node.tokens):
-        if token.kind == "MACRO":
+        if token.kind == "CMACRO":
```

Comment:
NOTE: This fix resulted in `_SPECIALIZE_UNPACK_SEQUENCE` becoming a viable uop. It's missing a `TIER_ONE_ONLY` marker; I'll add it back.